README.md

```diff
@@ -70,62 +75,39 @@ $ diesel migration run
 $ cargo build --release
 ```
 
-### Configure NEAR Indexer for Explorer
-
-To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
-
-```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
-```
+### Run NEAR Indexer for Explorer
 
-The above code will download the official genesis config and generate necessary configs. You can replace `testnet` in the command above to different network ID (`betanet`, `mainnet`).
+Command to run NEAR Indexer for Explorer have to include the chain-id and start options:
 
-These are the default config files that one could use just for the reference:
-  - [testnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/config.json)
-  - [betanet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/betanet/config.json)
-  - [mainnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json) 
+You can choose NEAR Indexer for Explorer start options:
+ - `from-latest` - start indexing blocks from the latest finalized block
+ - `from-interruption` - start indexing blocks from the block NEAR Indexer was interrupted last time but earlier for `<number_of_blocks>` if provided
+ - `from-block --height <block_height>` - start indexing blocks from the specific block height
 
-Configs for the specified network are in the `--home-dir` provided folder. We need to ensure that NEAR Indexer for Explorer follows
-all the necessary shards, so `"tracked_shards"` parameters in `~/.near/testnet/config.json` needs to be configured properly.
-For example, with a single shared network, you just add the shard #0 to the list:
+Unlike the original [NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer) you **can't** tell Indexer to store data from genesis (Accounts and Access Keys) by adding key `--store-genesis` to the `run` command. So please, ensure you took care about the genesis data in your database in order this indexer to work properly.
```

Comment:
Could you please create a tracking issue about the implementation of the store genesis feature and reflect the information in this paragraph instead?